### PR TITLE
feat: add marks

### PR DESCRIPTION
**Pull request by Yelinz**
marks are a special type of tags, which get their own icon and prominent toggle

### CODE REVIEW

**Comment by Yelinz** [comment]
@anehx do you have an idea how let the consumers define the used icons through alexandria for fontawsome?
These are some issues I looked at which hint at the same problem. https://github.com/FortAwesome/ember-fontawesome/issues/45 https://github.com/FortAwesome/ember-fontawesome/issues/8
Otherwise it might need to be configured just to include all the icons?

**Comment by anehx** [comment]
> @anehx do you have an idea how let the consumers define the used icons through alexandria for fontawsome? These are some issues I looked at which hint at the same problem. [FortAwesome/ember-fontawesome#45](https://github.com/FortAwesome/ember-fontawesome/issues/45) [FortAwesome/ember-fontawesome#8](https://github.com/FortAwesome/ember-fontawesome/issues/8) Otherwise it might need to be configured just to include all the icons?

I think that's what `config/icons.js` is used for.

File: addon/services/tags.js [hunk 1]
```diff
@@ -22,7 +22,29 @@ export default class TagsService extends Service {
   @lastValue("fetchAllTags") allTags;
 
   /** The searchTags are used in the TagFilter component. */
-  @lastValue("fetchSearchTags") searchTags;
+  // @lastValue("fetchSearchTags") searchTags;
```
Comment (czosel) [inline addon/services/tags.js:25]: drop?

**Comment by czosel** [comment]
Otherwise LGTM :+1:

**Review by czosel — APPROVED** [review]

**Pull request merged**